src/main/resources/liquibase/fixture-db/fixture-db.sql

```diff
@@ -19205,218 +19205,6 @@ UNLOCK TABLES;
 --
 -- Dumping routines for database 'ibdbv2_generic_merged'
 --
-/*!50003 DROP FUNCTION IF EXISTS `GetNumberInName` */;
```

Comment:
@mderamos this change should not be part of this branch.
Could you please check